src/components/manage/Toolbar/Types.jsx

```diff
@@ -32,6 +34,58 @@ const Types = ({ types, pathname }) => {
           ))}
         </ul>
       </div>
+      {settings.isMultilingual &&
+        (() => {
+          const translationsLeft = filter(
+            settings.supportedLanguages,
+            lang =>
+              !Boolean(
+                find(content['@components'].translations.items, {
+                  language: lang,
+                }),
+              ) && currentLanguage !== lang,
+          );
+          console.log(translationsLeft);
```

Comment:
Remove the console log

Reply:
I somehow recall adding an eslint check for console and excluding the node files from that. Will re-visit. Those things should get catched by the CI.

Reply:
I was surprised as well, the config is:

```
    "no-alert": 1,
    "no-debugger": 1,
```

Catch console statements is in other directive. However, it's kind of tricky because it will catch also really wanted `console` statements (eg. in the server part, in the error log, and in the console messages) that we will have to disable via `disable-line` comment statements.

Reply:
You can exclude console statements if you really need them (I did this in the server.jsx). Still odd why this wasn't caught by eslint.